app/api/templates/templates.ts

```diff
@@ -58,11 +66,11 @@ const updateTranslation = (currentTemplate, template) => {
 };
 
 export default {
-  async save(template, language) {
+  async save(template: TemplateSchema, language: string) {
     /* eslint-disable no-param-reassign */
     template.properties = template.properties || [];
-    /* eslint-disable no-param-reassign */
-    template.properties = generateNamesAndIds(template.properties);
+    template.properties = await generateNamesAndIds(template.properties);
+    /* eslint-enable no-param-reassign */
```

Comment:
isn't this rule enabled by default?

Reply:
@konz yes it is, in this chunk you can see on line 64 that it is disabled, thats why then is enabled again